presto-docs/src/main/sphinx/release/release-0.222.rst

```diff
@@ -0,0 +1,45 @@
+=============
+Release 0.222
+=============
+
+General Changes
+---------------
+* Fix incorrect results when dividing certain decimal numbers.
+* Fix query failure for certain join queries with "Equi criteria are empty,
+  so INNER join should not have PARTITIONED distribution type".
+* Add support for recoverable grouped execution. This can be enabled with the
+  ``recoverable_grouped_execution`` session property.
+* Add support for grouped execution for ScanFilterProject-only queries. This
```

Comment:
Is there a way to say this without referencing an internal operator name?

Reply:
How about `mapper only`? I originally thought about using mapper but then think we don't have such concept. Basically it means "no join and no aggregation".

Reply:
What about `single stage queries` (to use the presto concept)?

Reply:
But for something like `INSERT INTO <> AS SELECT * FROM foo` wouldn't we have 3 stages? 1) Scan 2) Write 3) Coordinator stats

Reply:
I guess that's true. what about `Add support for grouped execution for queries with no joins or aggregations`?

Reply:
Changed as per Rebecca's suggestion. Also added link to PR.